docs/content/manual/_incoming/2229-2449-nic-ip-vip-different-harvester-installer copy.md

```diff
@@ -0,0 +1,22 @@
+---
+title: NIC ip and vip can't be the same in Harvester installer
+---
+
+* Related issues: [#2229](https://github.com/harvester/harvester/issues/2229) [BUG] input nic ip and vip with same ip address in Harvester-Installer
+* * Related issues: [#2449](https://github.com/harvester/harvester/issues/2449) [backport v1.0] [BUG] input nic ip and vip with same ip address in Harvester-Installer
```

Comment:
Should be only one `*`

Reply:
Thanks for the check, remove the duplicate `*`